components/com_joomgallery/views/image/view.raw.php

```diff
@@ -226,53 +226,78 @@ public function display($tpl = null)
     JResponse::setHeader('Content-disposition', $disposition.'; filename='.basename($img));
 
     // Inlude watermark and crop
-    if(($include_watermark || $crop_image) && !$model->isGif($img))
+    // Create tmp file
+    $tmp_folder = JFactory::getApplication()->get('tmp_path');
```

Comment:
What do you think is preferable to use, the Joomla-Core tmp-folder or the JoomGallery tmp-folder?